cmd/helpMessages.go

```diff
@@ -134,162 +160,176 @@ const jobsCmdLongDescription = "Sub-commands related to managing jobs."
-  - azcopy rm "https://[account].dfs.core.windows.net/[container]/[path/to/file]?[SAS]"
+   - azcopy rm "https://[account].blob.core.windows.net/[container]/[path/to/parent/dir]" --recursive=true --list-of-files=/usr/bar/list.txt
+   - file content:
+	   dir1/dir2
+	   blob1
+	   blob2
+
+Remove a single file from a Blob Storage account that has a hierarchical namespace (include/exclude not supported):
 
-Remove a single directory from ADLS Gen2 (include/exclude not supported):
-  - azcopy rm "https://[account].dfs.core.windows.net/[container]/[path/to/directory]?[SAS]"
+   - azcopy rm "https://[account].dfs.core.windows.net/[container]/[path/to/file]?[SAS]"
+
+Remove a single directory from a Blob Storage account that has a hierarchical namespace (include/exclude not supported):
+
+   - azcopy rm "https://[account].dfs.core.windows.net/[container]/[path/to/directory]?[SAS]"
 `
 
 // ===================================== SYNC COMMAND ===================================== //
 const syncCmdShortDescription = "Replicate source to the destination location"
 
 const syncCmdLongDescription = `
-Replicate a source to a destination location. The last modified times are used for comparison, the file is skipped if the last modified time in the destination is more recent. The supported pairs are:
+The last modified times are used for comparison. The file is skipped if the last modified time in the destination is more recent. The supported pairs are:
+  
   - local <-> Azure Blob (either SAS or OAuth authentication can be used)
   - Azure Blob <-> Azure Blob (Source must include a SAS or is publicly accessible; either SAS or OAuth authentication can be used for destination)
   - Azure File <-> Azure File (Source must include a SAS or is publicly accessible; SAS authentication should be used for destination)
 
-Please note that the sync command differs from the copy command in several ways:
-  0. The recursive flag is on by default.
-  1. The source and destination should not contain patterns(such as * or ?).
-  2. The include/exclude flags can be a list of patterns matching to the file names. Please refer to the example section for illustration.
-  3. If there are files/blobs at the destination that are not present at the source, the user will be prompted to delete them. This prompt can be silenced by using the corresponding flags to automatically answer the deletion question.
-  4. When syncing between virtual directories, add a trailing slash to the path (refer to examples) if there's a blob with the same name as one of the virtual directories 
+The sync command differs from the copy command in several ways:
+
+  1. Sync can copy directories without the recursive flag, but it will not sync subdirectories.
+  2. The source and destination should not contain patterns (For example: * or ?).
+  3. The include-pattern and exclude-pattern flags can be a list of patterns matching to the file names. Please refer to the example section for illustration.
```

Comment:
I'm not sure what's happend with point #3 here.  The new #3 is not a difference compared to copy, because copy now has the exact same parameters.  And the old #3, about deletion, is important and still (almost) true.  I think the correct wording would be:

"If the deleteDestination flag is set to true or prompt, then sync will delete files/blobs at the destination that are not present at the source."

Reply:
I'm not sure what you mean about "old number 3". I do remember there being a statement about delete and being prompted. The statement was this "If there are files or blobs at the destination that aren't present at the source, the user will be prompted to delete them.". Adele stated that this was inaccurate so I removed it. I *think* you're asking me to add a new bullet with the string "If the deleteDestination flag is set to true or prompt, then sync will delete files/blobs at the destination that are not present at the source.". I'll proceed with that to unblock the PR.  Thx

Reply:
Correct.